components/dependency_injection.rst

```diff
@@ -302,6 +302,25 @@ config files:
             ->register('newsletter_manager', 'NewsletterManager')
             ->addMethodCall('setMailer', [new Reference('mailer')]);
 
+    .. code-block:: php-fluent-di
+
+        use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
```

Comment:
Should be `namespace Symfony\Component\DependencyInjection\Loader\Configurator;`

Reply:
kk, will take a look.

Reply:
I think this is a basic and deserves an addition in all example :). What do others think?

Reply:
OK, @HeahDude I just checked, and my documentation is right. 

`Symfony\Component\DependencyInjection\Loader\Configurator;` isn't an actual class, it's `ContianerConfigurator` like mine shows. Also - https://symfony.com/blog/new-in-symfony-3-4-php-based-configuration-for-services-and-routes shows ContainerConfigurator too.

And I also checked my 3 sites that use the fluent di configuration and they all use ContainerConfigurator as well.

Reply:
Maybe you misread my comment, I mean you should add the namespace and remove all use statements. It would work the same and would be cleaner. We do this for routing too.

Reply:
Ah, I definitely did mis-understand your comment. 

So are you asking that I update every place where I import `use function ...` and replace an example function call to `ref('id')` with something like `Configurator\ref('id')`?

if so, how would you feel about doing something like:

```
use Symfony\Component\DependencyInjection\Loader\Configurator as di;

Reply:
Also, for precedent, when importing symfony validation constraints, the standard is to alias `Symfony\Component\Validation\Constraints as Assert` to make the dx better for using the constraints.

Reply:
Using the namespace should prevent the need of any use statement:
```php
namespace Symfony\Component\DependencyInjection\Loader\Configurator;

return function(ContainerConfigurator $configurator) {
    // ...
    $container->set('svc')->args([ref('abc')];
};
```

Reply:
Ai, ok, that makes sense, will update across the board.

Reply:
Thank you!